src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveLoansController.java

```diff
@@ -139,6 +139,30 @@ public String addLoan(@PathVariable String companyNumber,
                 URI.expand(companyNumber, transactionId, companyAccountsId).toString();
     }
 
+    @GetMapping("remove/{loanId}")
+    public String deleteLoan(@PathVariable String companyNumber,
+                             @PathVariable String transactionId,
+                             @PathVariable String companyAccountsId,
+                             @PathVariable String loanId,
+                             @ModelAttribute(ADD_OR_REMOVE_LOANS) AddOrRemoveLoans addOrRemoveLoans,
```

Comment:
Is this used? I think it'd be null anyway as there's no form submission to trigger this method

Reply:
well spotted, i've removed this!